pandas/io/formats/format.py

```diff
@@ -1597,16 +1599,15 @@ def __init__(
         self.nat_rep = nat_rep
         self.box = box
 
-    def _format_strings(self) -> ndarray:
+    def _format_strings(self) -> List[str]:
         formatter = self.formatter or _get_format_timedelta64(
             self.values, nat_rep=self.nat_rep, box=self.box
         )
-        fmt_values = np.array([formatter(x) for x in self.values])
-        return fmt_values
+        return [formatter(x) for x in self.values]
```

Comment:
this is the most relevant change here.

return type now consistent with parent class and other formatters.

mypy should have picked this up. but since we don't have numpy stubs, all numpy types resolve to Any